### DISCUSSION TITLE
Load Bitmap Image into DrawingView

### DISCUSSION THREAD

**Discussion by devcrux** [CommunityToolkit/Maui#1288] [New Feature Discussions]
There is a common use case of drawing over an existing image to create some sort of annotation or highlight.

There is no way to do this currently using the DrawingView.

**Comment by VladislavAntonyuk** [comment]
You have an access to canvas, so you can draw any content. You can see MauiSamples/MauiPaint sample

**Reply by devcrux** [reply]
I dug a bit deeper and I saw the Canvas is inside the DrawAction.

Putting this here for anyone who may need it.
```
myDrawView.DrawAction = (canvas, rect) =>
{
        canvas.DrawSomething();
};
```
